www/app/App.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import BaseApp from 'next/app';
+import Head from 'next/head';
+import keyboardOnlyOutlines from 'keyboard-only-outlines';
+import ReactGA from 'react-ga';
+import getConfig from 'next/config';
+
+import seoImage from '../shared/media/images/seo.png';
+import favicon from '../shared/media/favicons/favicon.ico';
+
+import '../shared/styles/index.css';
+
+const { publicRuntimeConfig } = getConfig();
+
+const SEO_DATA = {
+    url: 'https://{project-domain}',
+    title: '{project-name}',
+    description: '{project-description}',
+    keywords: ['{project-keyword-1}', '{project-keyword-2}'],
+    image: { src: seoImage, width: 1200, height: 630 },
+};
+
+class App extends BaseApp {
```

Comment:
I would use `export class App extends NextApp {` for two reasons:

- Calling it `NextApp` instead of `BaseApp` it's easier for someone new to understand we are extending something from Next.js
- Exporting it so we can unit test the component instead of the wrapped component (also we should have tests already)